lib/project_types/script/layers/infrastructure/languages/task_runner.rb

```diff
@@ -15,9 +17,36 @@ def self.for(ctx, language, script_name)
           end
 
```

Comment:
I think this class should also be an interface for future language task runners. We typically do this by adding each method that needs to be implemented with a skeleton definition that the child class would overwrite. Like

```ruby
def build
  raise NotImplementedError
end
```

This way, it's easy for sub classes to see what they need to implement, and it's easy to catch errors if they forget to implement it.